Europe/es_ES.json

```diff
@@ -0,0 +1,261 @@
+  "No_data_found,_insert/select_some_data_to_see_visualizations_here.": "No se han encontrado datos, inserte/seleccione algunos datos para ver visualizaciones aquí.",
+  "Edit_Odometer_Record": "Editar registro de cuentakilómetros",
+  "Import_Data_from_CSV": "Importar datos desde CSV",
+  "In_order_for_this_utility_to_function_properly,_your_CSV_file_MUST_be_formatted_exactly_like_the_provided_sample._Dates_must_be_supplied_in_a_string._Numbers_must_be_supplied_as_numbers_without_currency_formatting.": "Para que esta utilidad funcione correctamente, su archivo CSV DEBE tener el mismo formato que la muestra proporcionada. Las fechas deben suministrarse en forma de cadena. Los números deben suministrarse como números sin formato de moneda.",
+  "Failure_to_format_the_data_correctly_can_cause_data_corruption._Please_make_sure_you_make_a_copy_of_the_local_database_before_proceeding.": "Si no se formatean correctamente, los datos pueden corromperse. Asegúrate de hacer una copia de la base de datos local antes de continuar.",
+  "Download_Sample": "Descargar ejemplo",
+  "Upload_CSV_File": "Cargar archivo CSV",
+  "Import": "Importar",
+  "Edit_Service_Record": "Editar registro de servicio",
+  "Date_service_was_performed": "Fecha en que se realizó el servicio",
+  "Odometer_reading_when_serviced": "Lectura del cuentakilómetros en el momento del mantenimiento",
+  "Description_of_item(s)_serviced(i.e._Oil_Change)": "Descripción del servicio (por ejemplo, cambio de aceite)",
+  "Cost_of_the_service": "Coste del servicio",
+  "Move_To": "Mover a",
+  "#_of_Repair_Records": "N.º de registros de reparación",
+  "Add_Repair_Record": "Añadir registro de reparación",
+  "Add_New_Repair_Record": "Añadir nuevo registro de reparación",
+  "Date_repair_was_performed": "Fecha de la reparación",
+  "Odometer_reading_when_repaired": "Lectura del cuentakilómetros una vez reparado",
+  "Description_of_item(s)_repaired(i.e._Alternator)": "Descripción de los elementos reparados (por ejemplo, el alternador)",
+  "Cost_of_the_repair": "Coste de la reparación",
+  "Choose_Supplies": "Elegir suministros",
+  "Add_Reminder": "Añadir recordatorio",
+  "Select_Supplies": "Seleccionar suministros",
+  "No_supplies_with_quantities_greater_than_0_is_found.": "No se han encontrado suministros con cantidades superiores a 0.",
+  "Select": "Seleccionar",
+  "#_of_Upgrade_Records": "N.º de registros de actualización",
+  "Add_Upgrade_Record": "Añadir registro de actualización",
+  "Add_New_Upgrade_Record": "Añadir nuevo registro de actualización",
+  "Date_upgrade/mods_was_installed": "Fecha de instalación de la actualización/modificación",
+  "Odometer_reading_when_upgraded/modded": "Lectura del cuentakilómetros cuando se actualiza/modifica",
+  "Description_of_item(s)_upgraded/modded": "Descripción de los elementos mejorados/modificados",
+  "Cost_of_the_upgrade/mods": "Coste de la actualización/modificación",
+  "#_of_Gas_Records": "N.º de registros de ga",
```

Comment:
abbreviation?

Reply:
First thank you for the review :+1: .

It is not an abbreviation, it is simply a mistake. Sorry.

...but it's good for me to hear a second opinion.

In the original translation, the word `gas` is used, but I think I understand that it actually refers to the word `fuel` (`combustible` in Spanish).

Did I misunderstand?

Reply:
No problem :)

Yes, translating to "combustible" it's ok